Makefile

```diff
@@ -15,7 +15,7 @@ mypy:
 	mypy openslides_backend/ tests/
 
 test:
-	pytest
+	pytest -s
```

Comment:
Why this?

Reply:
I want to see printlines, so I can debug it.